sourcegraph/about
# Rm support page and redirect to contact page

This closes #5757 

### Changelog
- remove support page
- redirect /support to /contact

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure /support redirects to /contact
